site2/docs/client-libraries.md

```diff
@@ -4,42 +4,74 @@ title: Pulsar client libraries
-| Node.js   | [User doc](client-libraries-node.md)                                          | [Here](https://github.com/apache/pulsar-client-node/releases)            | [Here](https://github.com/apache/pulsar-client-node)                          |
-| C#        | [User doc](client-libraries-dotnet.md)                                        | [Here](https://github.com/apache/pulsar-dotpulsar/blob/master/CHANGELOG) | [Here](https://github.com/apache/pulsar-dotpulsar)                            |
+| Language  | Documentation                          | Release note                                                                   | Code repo                                                                        |
+| --------- | -------------------------------------- | ------------------------------------------------------------------------------ | -------------------------------------------------------------------------------- |
+| Java      | [User doc](client-libraries-java.md)   | [Bundled](/release-notes/)                                                     | [Bundled](https://github.com/apache/pulsar/tree/master/pulsar-client)            |
+| C++       | [User doc](client-libraries-cpp.md)    | [Bundled](/release-notes/)                                                     | [Bundled](https://github.com/apache/pulsar/tree/master/pulsar-client-cpp)        |
+| Python    | [User doc](client-libraries-python.md) | [Bundled](/release-notes/)                                                     | [Bundled](https://github.com/apache/pulsar/tree/master/pulsar-client-cpp/python) |
+| Go client | [User doc](client-libraries-go.md)     | [Standalone](https://github.com/apache/pulsar-client-go/blob/master/CHANGELOG) | [Standalone](https://github.com/apache/pulsar-client-go)                         |
+| Node.js   | [User doc](client-libraries-node.md)   | [Standalone](https://github.com/apache/pulsar-client-node/releases)            | [Standalone](https://github.com/apache/pulsar-client-node)                       |
+| C#        | [User doc](client-libraries-dotnet.md) | [Standalone](https://github.com/apache/pulsar-dotpulsar/blob/master/CHANGELOG) | [Standalone](https://github.com/apache/pulsar-dotpulsar)                         |
 
-:::note
+Pulsar supports the following language-agnostic client libraries:
 
-- The code repos of **Java, C++, Python,** and **WebSocket** clients are hosted in the [Pulsar main repo](https://github.com/apache/pulsar) and these clients are released with Pulsar, so their release notes are parts of [Pulsar release note](/release-notes/).
-- The code repos of **Go, Node.js,** and **C#** clients are hosted outside of the [Pulsar main repo](https://github.com/apache/pulsar) and these clients are not released with Pulsar, so they have independent release notes.
-
```

Comment:
This note is included as "bundled" or "standalone".

Reply:
Suggest keeping lines 21-22. Reasons:

1. It's more clear to show users the whole information (lines 21-22), especially for those who are not familiar with Pulsar.

2. Some users can not fully understand the meaning of `standalone` and `bundled` because they can represent other meanings as well, or at least, they can not understand it at first glance and need to take a little more time to do the guesswork.

🔹🔹🔹

So how about compromising it as:

- Keep `standalone` and `bundled` in tables
- Also keep the whole explanation (lines 21 -22)

In this way, there is no redundant, missing, or confusing info.

Reply:
> Also keep the whole explanation (lines 21 -22)

Concise is important to document. We don't have to write down all information.

Users don't think of what the word means at first but click the link, and the "standalone" result doesn't confuse. While the "bundled" result is explained by its result - it's along with the main repo / main release.

Reply:
In another word, users don't care about how we organize release note and codebase, if only the link directs to the right place.